lib/kira/projects/queries/issue.ex

```diff
@@ -8,8 +8,44 @@ defmodule Kira.Projects.Queries.IssueQueries do
 
   alias Kira.Projects.Entities.Issue
 
+  @doc """
+  Gets issue by third-party id.
+  """
   def get_issue!(uid) do
     Issue
     |> Repo.get_by!(uid: uid)
   end
+
+  @doc """
+  Selects open issues.
+  """
+  def opened(query) do
+    query |> where([i], i.state == "opened")
+  end
+
+  @doc """
+  Selects issues for particular project.
+  """
+  def for_project(query, project) do
+    from i in query,
+      join: p in assoc(i, :project),
+      where: p.id == ^project.id
+  end
+
+  @doc """
+  Selects issues with no assignee.
+  """
+  def with_no_assignee(query) do
+    query |> where([i], is_nil(i.assignee_id))
+  end
+
+  @doc """
+  Selects an issue with the highest priority.
+  """
+  def with_highest_priority(query) do
+    # TODO: implement prioritizing
```

Comment:
Can you please create a new task for it?

Reply:
Based on the discussion of the way to prioritize, I just implemented ordering by weight and covered it by a test just in this PR